apps/etterlatte-saksbehandling-ui/client/src/components/klage/oppsummering/KlageOppsummering.tsx

```diff
@@ -1,43 +1,193 @@
-import { BodyShort, Button, Heading } from '@navikt/ds-react'
-import React from 'react'
+import { Alert, BodyShort, Button, Heading, Modal } from '@navikt/ds-react'
+import React, { useCallback, useEffect, useRef } from 'react'
```

Comment:
React

Reply:
Jeg tror egentlig ikke jeg bryr meg om den importen er med eller ikke, så lenge vi ikke har noen automatisk måte å få det ene eller det andre, eller noe warnings for å eksplisitt importere det.